README.md

```diff
@@ -396,6 +396,259 @@ public CloudTasksClient getService() throws IOException {
 }
 ```
 
+## Retries
+Client Libraries use retries to handle unexpected, transient failures (i.e. Server temporarily unavailable).
+Multiple attempts, hopefully, will result in a successful response from the server.
+
+By default, retries are configured by the Cloud Service. The configured retry parameters are defined per RPC.
+A service *may* choose to only enable retries for a subset of RPCs. It is possible that for a single service, 
+each RPC is configured differently.
+
+### Google Cloud Client Library Retry Concepts
+Enabling retries allow an RPC multiple attempts to try and achieve a successful call. A successful call
+is a response from a server that returns an `OK` Status Code (from gRPC) or a `2xx` Status Code (from HttpJson).
+
+An RPC will be retried when _both_ of the following scenarios occur:
+- Non-successful status code is received by the library and the status code is marked as retryable*
+- An RPC invocation exceeds the individual RPC bounds, but still falls within total RPC bounds**
```

Comment:
How about listing the terms such as "bounds" (or "RPC bounds")?

Reply:
Sounds good, added a blurb above to describe it.